Dockerfile

```
@@ -113,7 +113,7 @@ ENV PATH="${PATH}:/home/renovate/python3.13/bin"
 WORKDIR /home/renovate/renovate
 
 # Clone Renovate from specific ref (that includes the RPM lockfile support)
-RUN git clone --depth=1 --branch rpm-lockfiles-new https://github.com/redhat-exd-rebuilds/renovate.git .
+RUN git clone --depth=1 --branch rpm-lockfiles-new-container-cve https://github.com/redhat-exd-rebuilds/renovate.git .
```

```suggestion
RUN git clone --depth=1 --branch develop https://github.com/redhat-exd-rebuilds/renovate.git .
```

This is the new default branch, it's currently on the same commit as `rpm-lockfiles-new-container-cve`.